static/app/views/releases/thresholdsList/index.spec.tsx

```diff
@@ -0,0 +1,88 @@
+import {Organization} from 'sentry-fixture/organization';
+
+import {initializeOrg} from 'sentry-test/initializeOrg';
+import {render, waitFor} from 'sentry-test/reactTestingLibrary';
+
+import PageFiltersStore from 'sentry/stores/pageFiltersStore';
+import ThresholdsList from 'sentry/views/releases/thresholdsList/';
+
+describe('ReleaseThresholdsList', () => {
+  const organization = Organization({
+    slug: 'test-thresholds',
+    features: ['release-ui-v2'],
+  });
+
+  const {router, routerContext} = initializeOrg({
+    organization,
+  });
+
+  let mockThresholdFetch;
+
+  beforeEach(() => {
+    mockThresholdFetch = MockApiClient.addMockResponse({
+      url: `/organizations/${organization.slug}/release-thresholds/`,
+      method: 'GET',
+      body: [],
+    });
+    PageFiltersStore.init();
+  });
+
+  afterEach(() => {
+    PageFiltersStore.reset();
+    MockApiClient.clearMockResponses();
+  });
+
+  it('redirects to releases if flag is not set', () => {
+    const organization2 = Organization({
+      slug: 'test-thresholds-no-flag',
+      features: [],
+    });
+
+    const {router: flaglessRouter, routerContext: flaglessRouterContext} = initializeOrg({
+      organization: organization2,
+    });
+    render(<ThresholdsList />, {
+      context: flaglessRouterContext,
+      organization: organization2,
+    });
+
+    expect(flaglessRouter.replace).toHaveBeenCalledTimes(1);
+    expect(flaglessRouter.replace).toHaveBeenCalledWith(`/releases/`);
+  });
+
+  it('fetches release thresholds for selected projects', async () => {
+    const query = {
+      project: [-1],
+    };
+    render(<ThresholdsList />, {context: routerContext, organization});
+    await waitFor(() =>
+      expect(mockThresholdFetch).toHaveBeenCalledWith(
+        '/organizations/test-thresholds/release-thresholds/',
+        expect.objectContaining({query})
+      )
+    );
```

Comment:
wait for something from the request to be rendered from the component which helps make sure the component doesn't error on render and lets you remove the waitFor

```suggestion
			expect(await screen.findByText('something').toBeInTheDocument();
      expect(mockThresholdFetch).toHaveBeenCalledWith(
        '/organizations/test-thresholds/release-thresholds/',
        expect.objectContaining({query})
      )
```